.github/workflows/ci.yml

```diff
@@ -14,6 +14,25 @@ jobs:
       run: sudo apt-get install autoconf automake pkg-config libevent-dev libpcre3-dev libssl-dev
     - name: Build
       run: autoreconf -ivf && ./configure && make
+    - name: Install test dependencies
+      run: |
+        git clone git://github.com/antirez/redis.git --branch 5.0.7
+        cd redis && make malloc=libc && make && cd ..
```

Comment:
@filipecosta90 Any particular reason you use `malloc=libc`?

Reply:
For this purpose none. We compile with malloc=libc on other CI's since we might want to use valgrind to check for leaks on the modules. On this case no real usage. changing towards default jemalloc